### PR TITLE
V6/fix swap btn

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Add swap button
   - Replace existing swap button with new one

Screenshot from 2021-05-27 19-05-27
Screenshot from 2021-05-27 19-05-43

### CODE REVIEW

**Comment by olgakup** [comment]
Please in the future do not mix features when making a pr. ENS icons could have been based of the develop branch and merged sooner

**Comment by olgakup** [comment]
Please fix the build to get full review

File: src/dapps/ens-manager-dapp/TheENSManagerLayout.vue [hunk 1]
```diff
@@ -160,25 +160,26 @@
...
+                      class="text-center cursor-pointer"
+                      @click="manage(option.type, idx)"
                     >
-                      <div>
-                        {{
-                          $t('ens.manage-domains.expire-at', {
-                            date: domain.expiration
-                          })
-                        }}
+                      <v-icon color="titlePrimary" x-large>{{
```
Comment (olgakup) [inline src/dapps/ens-manager-dapp/TheENSManagerLayout.vue:168]: can you change this to mew green color

**Review by olgakup — CHANGES_REQUESTED** [review]

File: src/dapps/ens-manager-dapp/TheENSManagerLayout.vue (hunk 1 above)
Comment (hsurf22) [reply src/dapps/ens-manager-dapp/TheENSManagerLayout.vue]: fixed

**Pull request merged**